nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/LogAttribute.java

```diff
@@ -45,6 +45,8 @@
 
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang3.StringUtils;
+import org.bouncycastle.util.Strings;
```

Comment:
This is unused and is causing errors in the checkstyle.  (Checkstyle can be evaluated via Maven Profile contrib-check, e.g. mvn <goals> -P contrib-check)